api/lib/mno_enterprise/csv_importer.rb

```diff
@@ -59,19 +59,22 @@ def self.process(file_path)
 
         # Create or Update User
         user = MnoEnterprise::User.where(email: row['email']).first
-        if user
-          report[:users][:updated] << user
-        else
-          user = MnoEnterprise::User.new
-          report[:users][:added] << user
-          user.password = Devise.friendly_token
-          user.email = row['email']
-        end
+        event_type = if user
+                       :added
```

Comment:
Shouldn't it be `:updated` in this case.
I'm a bit confused with this part of the code.

Reply:
Well spotted.